src/types/mod.rs

```diff
@@ -85,6 +85,30 @@ mod url;
 mod value;
 mod value_ref;
 
+/// Wrapper struct for timestamp, can be used for appending timestamp in appender
+#[derive(Copy, Clone)]
+pub enum Timestamp {
+    /// Second
+    Second(i64),
+    /// Millisecond
+    Millisecond(i64),
+    /// Microsecond
+    Microsecond(i64),
+    /// Nanosecond
+    Nanosecond(i64),
+}
+
+impl ToSql for Timestamp {
```

Comment:
Seems it's more reasonable to `impl ToSql for std::time::Duration` or `impl ToSql for std::time::Instant` ?

https://doc.rust-lang.org/std/time/struct.Instant.html
https://doc.rust-lang.org/std/time/struct.Duration.html

Reply:
And we'd better move the logic into https://github.com/wangfenjin/duckdb-rs/blob/main/src/types/to_sql.rs#L204

Reply:
The reason is if we define our own Timestamp type, users need to convert their type to ours